### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by JuanLPalacios**
### Implemented Features

- tables:
  - users
  - posts
  - comments
  - likes
- foreign keys
- indexes
- data schema

### CODE REVIEW

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @JuanLPalacios, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description looks professional✔️
- Descriptive commit messages✔️

## Required Changes ♻️
_Check the comments under the review._

- [x] Kindly update the readme file, the readme file is the first place user's learn about your project. Here is a [template ](https://github.com/microverseinc/readme-template/blob/master/README.md) from Microverse which you can customize to suit your need.

- [x] Let's enable linter actions, I can see you have configured rubocop. Let's check all the possible failures until we have a working linter action.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JuanLPalacios** [comment]
@elisha2kyakpo1 done with the readme.

regarding the enabling of linter actions, they are both enabled and set up, but are down at the moment [guthub services status](https://www.githubstatus.com/)
www githubstatus com_(desktop wide)

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      - name: Stylelint Report
+        run: npx stylelint "**/*.{css,scss}"
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (JuanLPalacios) [inline .github/workflows/linters.yml:44]: here you can see the linter setup

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @JuanLPalacios,

✔️ Good documentation

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**